AutomatticTracks/src/main/java/com/automattic/android/tracks/crashlogging/CrashLoggingDataProvider.kt

```diff
@@ -28,6 +28,12 @@ interface CrashLoggingDataProvider {
      */
     val enableCrashLoggingLogs: Boolean
 
+    /**
+     * Provides [CrashLogging] with information about exception that should be dropped if is the
+     * last one on stacktrace
+     */
+    val toDropIfLastException: CrashLoggingException?
```

Comment:
1. 🔍 Nitpick: I think it'd be good to add an example into the comment why the client app might want to do that. I think the EventBus might be a good example, wdyt?

2.❓Question: Have you considered using a function instead of a field here? Something like:
`shouldDropWrappingException(module: String, type: String, value: String): Boolean`

I think it might be a bit more extensible since the client could drop as many different wrappers as they wanted. Wdyt?

Reply:
> the client could drop as many different wrappers as they wanted

Totally agree, I'll change to a function. Thanks!